exporters/otlp/common/src/main/java/io/opentelemetry/exporter/internal/otlp/logs/LogStatelessMarshaler.java

```diff
@@ -40,13 +55,22 @@ public void writeTo(Serializer output, LogRecordData log, MarshalerContext conte
       output.serializeMessageWithContext(
           LogRecord.BODY, log.getBodyValue(), AnyValueStatelessMarshaler.INSTANCE, context);
     }
-    output.serializeRepeatedMessageWithContext(
-        LogRecord.ATTRIBUTES,
-        log.getAttributes(),
-        AttributeKeyValueStatelessMarshaler.INSTANCE,
-        context);
-    int droppedAttributesCount = log.getTotalAttributeCount() - log.getAttributes().size();
-    output.serializeUInt32(LogRecord.DROPPED_ATTRIBUTES_COUNT, droppedAttributesCount);
+    if (INCUBATOR_AVAILABLE) {
+      IncubatingUtil.serializeExtendedAttributes(output, log, context);
+
+      int droppedAttributesCount =
+          log.getTotalAttributeCount() - IncubatingUtil.extendedAttributesSize(log);
+      output.serializeUInt32(LogRecord.DROPPED_ATTRIBUTES_COUNT, droppedAttributesCount);
+    } else {
+      output.serializeRepeatedMessageWithContext(
+          LogRecord.ATTRIBUTES,
+          log.getAttributes(),
+          AttributeKeyValueStatelessMarshaler.INSTANCE,
+          context);
+
+      int droppedAttributesCount = log.getTotalAttributeCount() - log.getAttributes().size();
+      output.serializeUInt32(LogRecord.DROPPED_ATTRIBUTES_COUNT, droppedAttributesCount);
```

Comment:
nit: prefer to declare the int above the if/else and move the dropped count serialization out of the if/else (to below it).